hotel.json

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
@Kumjami do we need this sample hotel? it should have a `url` key/value in it to be valid.